Client/game_sa/CVehicleSA.cpp

```diff
@@ -489,6 +524,26 @@ void CVehicleSA::SetTrainSpeed(float fSpeed)
     pInterface->m_fTrainSpeed = fSpeed;
 }
 
+float CVehicleSA::GetHeliRotorSpeed()
+{
+    auto* heliInterface = static_cast<CHeliSAInterface*>(GetInterface());
+    return heliInterface->m_wheelSpeed[1];
+}
+
+void CVehicleSA::SetHeliRotorSpeed(float speed)
```

Comment:
Noexcept maybe?

Reply:
Theoretically, there is little chance of an exception if an access violation occurs for some reason. I think it's better to leave it as is in this case

Reply:
Wouldnt it crash either way in that case then?

Reply:
Adding noexcept will cause the game to terminate immediately without generating a standard error / crash window, which can make diagnosing the problem more difficult. In this case, if for some reason the value cannot be changed and an AVE occurs, having no noexcept might produce an error that helps identify the cause of the issue. 

Additionally, functions that interact with game memory should generally not be marked as noexcept. Memory can become invalid or corrupted for various reasons, and the chance of an exception always exists. Therefore, it's better to leave this function without noexcept to ensure that potential issues are properly reported and can be diagnosed